e2e/utils.ts

```diff
@@ -196,7 +199,14 @@ export const prepareStandaloneSetup = (fixtureName: string) => {
           'utf-8',
         ),
       );
-      const pnpmOverrides = rootPkg.pnpmOverrides;
+      const pnpmOverrides = {
+        ...rootPkg.pnpmOverrides,
+        waku: wakuPackageTgz,
+        react: '0.0.0-experimental-197d6a04-20250424',
+        'react-dom': '0.0.0-experimental-197d6a04-20250424',
+        'react-server-dom-vite':
+          'https://github.com/hi-ogawa/vite-plugins/raw/refs/heads/04-24-refactor_rsc_use_react-server-dom-vite/react-server-dom-vite-19.1.0.tgz',
+      };
```

Comment:
I just wonder if this works:

```suggestion
      const pnpmOverrides = {
        ...rootPkg.pnpm?.overrides,
        ...rootPkg.pnpmOverrides,
```

Oh, `waku: wakuPackageTgz,` is interesting.

Reply:
Hmmm, now I wonder if `rootPkg.pnpmOverrides` is a typo of `rootPkg.pnpm?.overrides`. @himself65

Reply:
#1417 is ready. It won't solve the problem in this PR, but it might make it a little cleaner.

Reply:
I modified #1417 and merged it. It's related to the problem in this PR, so please merge `main` and see how it goes.